backend/store/metric.go

```diff
@@ -0,0 +1,34 @@
+package store
+
+import (
+	"context"
+	"strings"
+)
+
+// CountUsers counts the principal.
+func (s *Store) CountUsers(ctx context.Context, find *FindUserMessage) (int, error) {
+	tx, err := s.db.BeginTx(ctx, nil)
+	if err != nil {
+		return 0, FormatError(err)
+	}
+	defer tx.Rollback()
+
+	where, args := getUserFindQuery(find)
+	count := 0
+
+	if err := tx.QueryRowContext(ctx, `
```

Comment:
We only need to count principals as it's 1:1 mapping to member table. Btw, we should also kickout the system bot.